### PR TITLE
Update vendor and add Dockerfile

### PR DESCRIPTION
**Pull request by wallyqs**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM golang:1.11-alpine3.8 AS builder
```
Comment (derekcollison) [inline Dockerfile]: 3.10 is out, not sure its important though but does update Go version as well.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+RUN apk add --update git
+
+COPY . .
+
+RUN CGO_ENABLED=0 go build -v -a -tags netgo -installsuffix netgo -ldflags "-s -w" -o /latency-tests
+
+FROM alpine:3.8
+
+RUN apk add --update ca-certificates && apk add tcptraceroute
```
Comment (derekcollison) [inline Dockerfile]: use '--no-cache' as well here I think.

File: vendor/github.com/codahale/hdrhistogram/.travis.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+language: go
+go:
+  - 1.5
```
Comment (derekcollison) [inline vendor/github.com/codahale/hdrhistogram/.travis.yml:3]: Remove these I think. Just do 1.11 and 1.12 and maybe tip.

Comment (derekcollison) [reply vendor/github.com/codahale/hdrhistogram/.travis.yml:3]: nvm this is vendor, but maybe update the vendored stuff ;)

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: vendor/github.com/codahale/hdrhistogram/.travis.yml (hunk 3 above)
Comment (wallyqs) [reply vendor/github.com/codahale/hdrhistogram/.travis.yml:3]: hmm that repository seems archived 🤔 https://github.com/codahale/hdrhistogram

File: Dockerfile (hunk 2 above)
Comment (wallyqs) [reply Dockerfile]: ok updated these

File: vendor/github.com/codahale/hdrhistogram/.travis.yml (hunk 3 above)
Comment (derekcollison) [reply vendor/github.com/codahale/hdrhistogram/.travis.yml:3]: Interesting.. For now ok I guess.

File: Dockerfile (hunk 1 above)
Comment (derekcollison) [reply Dockerfile]: This is not critical.

**Pull request closed without merging**